packages/pg-v5/commands/credentials/destroy.js

```diff
@@ -15,7 +15,10 @@ async function run(context, heroku) {
   }
 
   let db = await fetcher.addon(app, args.database)
-  if (util.essentialPlan(db)) {
+
+  if (util.essentialNumPlan(db)) {
+    throw new Error('You can’t perform this operation on Essential-tier databases.')
+  } else if (util.legacyEssentialPlan(db)) {
```

Comment:
can we combine these?
```suggestion
  if (util.essentialNumPlan(db) || util.legacyEssentialPlan(db)) {
```

Reply:
yes combined those ^